File: https:/github.com/abhinavtripathi17/DSA-Leetcode.git/26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.cpp
// Last updated: 8/4/2026, 7:25:35 PM
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        int k = 1;
        for(int i = 1 ; i < n ; i++){
            if(nums[i] == nums[i-1]){
                continue;
            }
            nums[k] = nums[i];
            k++;
            
        }
        return k;
    }
};

File: https:/github.com/abhinavtripathi17/DSA-Leetcode.git/26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 8/4/2026, 7:25:43 PM
class Solution {
    public int removeDuplicates(int[] nums) {
        int i = 0;
        for(int k = 1 ; k < nums.length ; k++){
            if(nums[i] != nums[k]){
                i++;
                nums[i] = nums[k];
            }
        }
        return i+1;
    }
}